python/gui/dependencies.txt

```diff
@@ -17,4 +17,4 @@ channels
 channels-redis
 djangorestframework
 psycopg2-binary  # TODO: get source package in future. Note that psycopg2 cannot be used on Mac; psycopg2-binary must be used
-numpy
+numpy==1.26.4
```

Comment:
No need to be so specific with the version. We just want to ensure that it is below 2.
```suggestion
numpy~=1.0
```

Reply:
I'm strongly considering taking this out entirely.  I realized there really is no evidence at this point that there is a problem with version 2.0.0 for anything running inside the GUI stack.  The point was to match the default I was setting up, but even assuming we still went that route, that really isn't necessary.